src/site/WebController.php

```diff
@@ -18,6 +18,15 @@ protected function getRequiredStringParam(string $key): string {
     return $this->parameters[$key];
   }
 
+  protected function getStringParam(
+    string $key, string $default = ""
+  ): string {
```

Comment:
would prefer for this to return ?string and possibly rename to getOptionalStringParam